indexer/internal/metrics/metrics.go

```diff
@@ -58,10 +84,33 @@ func (m *Metrics) getBalances() (map[string]float64, error) {
 	return balances, nil
 }
 
-func (m *Metrics) recordMetrics(balances map[string]float64) (err error) {
+func (m *Metrics) getTxStatus(storage teleportcontract.Storage) (txStatus, error) {
```

Comment:
The func name is not related to argument: txStatus - storage. Pass LastPegoutTxID directly.